client/components/Display.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import Cards from './Cards.jsx'
+
+
+const Display = (props) => {
+  const cards = []
+  for (let i = 0; i < props.officials.length; i++) {
+    cards.push(<Card />)
```

Comment:
For cleaner code, it looks like it's best practice to use .map method when creating an array of jsx elements instead of a standard for loop.

Reply:
done! good to know